components/console-backend-service/internal/gqlschema/k8sNew.graphql

```diff
@@ -0,0 +1,138 @@
+scalar LimitType @goModel(model: "k8s.io/api/core/v1.LimitType")
+
+type ResourceLimits {
+  memory: String
+  cpu: String
+}
+
+# LIMIT RANGE
+type LimitRangeItem @goModel(model: "k8s.io/api/core/v1.LimitRangeItem") {
+  type: LimitType!
+  max: ResourceLimits!
+  default: ResourceLimits!
+  defaultRequest: ResourceLimits!
+}
+
+type LimitRangeSpec @goModel(model: "k8s.io/api/core/v1.LimitRangeSpec") {
+  limits: [LimitRangeItem!]!
+}
+type LimitRange @goModel(model: "k8s.io/api/core/v1.LimitRange") {
+  name: String!
+  spec: LimitRangeSpec!
+  json: JSON!
+}
+
+# RESOURCE QUOTA
+
+type ResourceQuotaHard {
+  limits: ResourceLimits!
+  requests: ResourceLimits!
+  pods: String!
+}
+
+type ResourceQuotaSpec @goModel(model: "k8s.io/api/core/v1.ResourceQuotaSpec") {
+  hard: ResourceQuotaHard!
+}
+type ResourceQuota @goModel(model: "k8s.io/api/core/v1.ResourceQuota") {
+  name: String!
+  spec: ResourceQuotaSpec!
+  json: JSON!
+}
+
+extend type Query {
+  limitRanges(namespace: String!): [LimitRange!]!
+  @HasAccess(
+    attributes: {
+      resource: "limitRange"
+      verb: "list"
+      apiGroup: ""
+      apiVersion: "v1"
+      namespaceArg: "namespace"
+    }
+  )
+
+  resourceQuotas(namespace: String!): [ResourceQuota!]!
+  @HasAccess(
+    attributes: {
+      resource: "resourcequotas"
+      verb: "list"
+      apiGroup: ""
+      apiVersion: "v1"
+      namespaceArg: "namespace"
+    }
+  )
+
+  # resourceQuotasStatus(namespace: String!): ResourceQuotasStatus!
+  # @HasAccess(
+  #   attributes: {
+  #     resource: "resourcequotas"
+  #     verb: "list"
+  #     apiGroup: ""
+  #     apiVersion: "v1"
+  #     namespaceArg: "namespace"
+  #   }
+  # )
```

Comment:
//

Reply:
I left it on purpose. This query has been removed completely but we may need to add it in the future. This would be a good start 😃